app/models/comment.rb

```diff
@@ -1,4 +1,16 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  validates :user_id, presence: true
+  validates :post_id, presence: true
+  validates :text, presence: true
+
+  after_create :update_post_comments_counter
```

Comment:
**OPTIONAL** - You have done well so far, but please use the `after_save` module instead of the `after_create` module to update the comments counter since we would only like to update the counter when the data is saved into the database. Please fix this 😄